lib/Controllers/list.js

```diff
@@ -68,6 +69,14 @@ List.prototype.fetch = function(req, res, context) {
         order.push([sortColumn, 'ASC']);
       }
     });
+    var allowedColumns = this.resource.sort.attributes || Object.keys(model.rawAttributes);
+    var columnNames = order.map(_.first);
+    var disallowedColumns = _.difference(columnNames, allowedColumns);
+    if(disallowedColumns.length) {
+      res.status(400);
+      res.json({error: 'Sorting is not allowed on the attributes: ' + disallowedColumns.join(', ') });
```

Comment:
ah I guess I haven't been very consistent, the 500 below returns "error". I guess I'm still of a mind that we should adopt a consistent error response type throughout the project. Thoughts?